arctic/_config.py

```diff
@@ -94,13 +94,6 @@ class FwPointersCfg(Enum):
 ARCTIC_ASYNC_NWORKERS = os.environ.get('ARCTIC_ASYNC_NWORKERS', 4)
 
 
-# -------------------------------
-# Flag used for indicating caching levels. For now just for list_libraries.
-# -------------------------------
-ENABLE_CACHE = not bool(os.environ.get('DISABLE_CACHE'))
```

Comment:
Probably the right call to move away from system properties here.  But make sure Ray knows.